src/main/java/com/eprosima/idl/context/Context.java

```diff
@@ -986,6 +982,8 @@ else if (flag == 3)
                                     includeFile.substring(0, includeFile.length() - 4).replace('.', '_');
                                     // Change back to "." for relative paths.
                                     dependency_to_add = dependency_to_add.replaceAll("__/", "../");
+                                    // Substitute "\\" with File separator.
+                                    dependency_to_add = dependency_to_add.replace('\\', '/');
```

Comment:
I think that the suggested change on the issue was on a previous line
```suggestion
                                    includeFile.substring(0, includeFile.length() - 4).replace('.', '_').replace('\\', '/'));
                                    // Change back to "." for relative paths.
                                    dependency_to_add = dependency_to_add.replaceAll("__/", "../");
```

Reply:
Well, I think its the same in this case, as in this recent version, a temporal object is created. Resolving